src/main/resources/META-INF/services/org.apache.lucene.codecs.PostingsFormat

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2017 Rocana
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is an SPI (service provider interface) config file so Lucene can look up
+# our custom postings format at runtime using just the fully qualified class name
+# here.
+#
+# If you have access to the Rocana JIRA, see ROCANA-8229.
+
+com.rocana.lucene.codec.v1.RocanaLucene50PostingsFormat
```

Comment:
I know it's kind of late in the game here but the `v1` in the package name has been making me nervous for a while and now I know why - you've tied both the RocanaLucene50PostingsFormat and RocanaCodec (and probably the PerFieldsPostingsFormat) to be in "v1" together, but it is very likely that we will need to change them independently. I can see us needing to change/update the RocanaCodec, but not the `RocanaLucene50PostingsFormat`, but you would have to move both to a `v2` package even though one didn't change.

I think it is better to follow the Lucene convention of leaving them in an unversioned package and put the version in the name.  We already have the version in the name, (so yay).  My vote is to do like Lucene and put the RocanaCodec these in a "codecs" package with no version.